src/main.js

```diff
@@ -99,6 +98,7 @@ function starFavorite() {
     for (var i = 0; i < cards.length; i++) {
       if (cards[i].id == idToStar) {
         cards[i].toggleStar()
+        cards[i].saveToStorage()
```

Comment:
So since we are calling saveToStorage here when the user favorites an idea and it gets saved to local storage, we no longer need to call it in the saveCard function. The savecard function's functionality is now only to check for user inputs, enable the save button, display cards and then clear input fields after. This is my understanding of it now, let me know if i'm wrong. If so, this makes total sense to me!